Sources/HAP/Base/Characteristic.swift

```diff
@@ -50,6 +52,11 @@ extension Characteristic {
         if let maxValue = maxValue { serialized["maxValue"] = maxValue }
         if let minValue = minValue { serialized["minValue"] = minValue }
         if let minStep = minStep { serialized["minStep"] = minStep }
+        if let validValuesRange = validValuesRange {
```

Comment:
Should use range otherwise array?

Reply:
@Bouke

Reply:
The specification notes regarding ranges the following:

> This descriptor shall be used when it is more efficient to specify a list of valid values as a range rather than individual values.

So, this looks fine to me.